cwms-data-api/src/main/java/cwms/cda/data/dto/forecast/ForecastSpec.java

```diff
@@ -1,30 +1,36 @@
 package cwms.cda.data.dto.forecast;
 
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlAttribute;
-import javax.xml.bind.annotation.XmlRootElement;
+import io.swagger.v3.oas.annotations.media.Schema;
 
-@XmlRootElement(name = "alias")
+import javax.xml.bind.annotation.*;
+import java.util.List;
+
+@XmlRootElement(name = "forecast-spec")
 @XmlAccessorType(XmlAccessType.FIELD)
 public class ForecastSpec {
 
-  @XmlAttribute
-  private Integer fcstSpecCode;
-
-  @XmlAttribute
-  private Integer officeCode;
+  @Schema(description = "Forecast Spec ID")
+  @XmlElement(name = "spec-id")
+  String specId;
 
-  @XmlAttribute
-  private String fcstSpecId;
+  @Schema(description = "Office ID that owns the time-series")
+  @XmlElement(name = "office-id")
+  String officeId;
 
-  @XmlAttribute
-  private Integer locationCode;
+  @Schema(description = "Location ID")
+  @XmlElement(name = "location-id")
+  String locationId;
 
-  @XmlAttribute
-  private Integer soruceEntity;
+  @Schema(description = "Source Entity ID")
+  @XmlElement(name = "source-entity-id")
+  String sourceEntityId;
 
+  @Schema(description = "Description of Forecast")
   @XmlAttribute
   private String description;
 
+  @Schema(description = "List of Time Series IDs belonging to this Forecast Spec")
+  @XmlAttribute(name = "time-series-ids")
+  private List<String> timeSeriesIDs;

```

Comment:
Should we do TimeSeriesIdentifier here over a list of strings? (I'm probably splitting hairs though.)

Reply:
I like TimeSeriesIdentifier more given that we can determine LRTS vs PRTS (though that'll be moot soon), but we'd also get interval offset and location time zone.